### ISSUE TITLE
Weekly running TorchDynamo on nn.Modules crawled from github (14k github models)

### ISSUE THREAD

**Issue by yanboliang** [pytorch/pytorch#93427]
We [hit 100% pass rates on 7k github models ](https://github.com/pytorch/torchdynamo/issues/156)back to June. As we are working to support more features(not using fallback), we want to understand the current status of the pass rate. I re-run and the updated number is:
```
TorchDynamo ParityBench:
          total  passing  score
projects   1091     1004  92.0%
tests      7370     7204  97.7%
```

We are working to fix these new bugs and moving to 100% again! I'll keep to update the pass rate in this thread.

How to repro:
* Checkout https://github.com/jansel/pytorch-jit-paritybench
* Batch run ```python main.py --backend eager```
* Adhoc run:
  * ```pytest ./generated/{filename}.py -k test_{n}``` (e.g, ```pytest ./generated/test_KunpengLi1994_VSRN.py -k test_002```)
  * ```python -e ./generated/{filename}.py --backend eager```

cc @ezyang @soumith @msaroufim @wconstab @ngimel @bdhirsh

**Comment by yanboliang** [comment]
```
TorchDynamo ParityBench:
          total  passing  score
projects   1481     1425  96.2%
tests     14301    14214  99.4%
```

**Comment by yanboliang** [comment]
```
dynamo eager ParityBench:
          total  passing  score
projects   1480     1430  96.6%
tests     14304    14233  99.5%
```

**Comment by yanboliang** [comment]
```
dynamo eager ParityBench:
          total  passing  score
projects   1444     1382  95.7%
tests     13733    13634  99.3%
```

**Comment by yanboliang** [comment]
```
dynamo eager ParityBench:
          total  passing  score
projects   1443     1381  95.7%
tests     13731    13632  99.3%
```

**Comment by yanboliang** [comment]
```
dynamo eager ParityBench:
          total  passing  score
projects   1444     1374  95.2%
tests     13730    13618  99.2%
```

**Comment by yanboliang** [comment]
This has been merged into #93667, please go there to check the pass rate across all backends.
